course/lesson-8-defining-variables/lesson.tres

```diff
@@ -253,7 +255,7 @@ text = "The above code is like taking a label from the appropriate item and stic
 - At line 2, the [code]health[/code] variable holds a number.
 - From line 3, [code]health[/code] holds text.
 
-The computer will let you do that! The code's syntax and \"grammar\" are correct, but it's not good.
+The computer will let you do that! The code's syntax and \"grammar\" are correct, but it's not good. Variable names should describe the value they contain, so a [code]health[/code] variable which has a text value will be confusing to your future self and other coders. It can also cause errors in your program.
```

Comment:
```suggestion
The computer will let you do that! The code's syntax and \"grammar\" are correct, but it's not good.

Variable names should describe the value they contain, so a [code]health[/code] variable with a text value will confuse your future self and other coders. It can also cause errors in your game.
```